testdriver/testdriver.go

```diff
@@ -805,8 +810,21 @@ func ErrLogMatch(ts *testscript.TestScript, neg bool, args []string) {
 		ts.Fatalf("errlogmatch: failed to parse args %v: %v", args, err)
 	}
 
-	if *fWait != "" && neg {
-		ts.Fatalf("-wait is not compatible with negating the command")
+	var waitSet bool
+	fs.Visit(func(f *flag.Flag) {
+		if f.Name == "wait" {
+			waitSet = true
+		}
+	})
+
```

Comment:
Really nits, would this be more readable?
```
if waitSet && neg {
        ts.Fatalf(...)
}

if *fWait == "" && !neg {
        fWait = &defaultWait
}
```

Reply:
It is, thanks.